plugins/database/cassandra/testoutput.json

```diff
@@ -0,0 +1,35 @@
e incorrect\n2020/10/12 11:31:05 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:31:08 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:31:20 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:20 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:21 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:22 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:24 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:25 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:31:29 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:31:37 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:37 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:38 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:39 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:41 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:31:44 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:31:48 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:31:52 gocql: unable to dial control conn 127.0.0.1: Provided username v_test_test_co7ceqbuubvcnjdnatwr_1602523911 and/or password are incorrect\n2020/10/12 11:31:52 gocql: unable to dial control conn 127.0.0.1: Provided username v_test_test_co7ceqbuubvcnjdnatwr_1602523911 and/or password are incorrect\n2020/10/12 11:31:53 gocql: unable to dial control conn 127.0.0.1: Provided username v_test_test_co7ceqbuubvcnjdnatwr_1602523911 and/or password are incorrect\n2020/10/12 11:31:53 gocql: unable to dial control conn 127.0.0.1: Provided username v_test_test_co7ceqbuubvcnjdnatwr_1602523911 and/or password are incorrect\n2020/10/12 11:31:56 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:57 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:57 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:31:58 gocql: unable to dial control conn 127.0.0.1: EOF\n2020/10/12 11:32:00 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:32:02 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:32:06 gocql: unable to dial control conn 127.0.0.1: Provided username cassandra and/or password are incorrect\n2020/10/12 11:32:12 gocql: unable to dial control conn 127.0.0.1: v_test_test_i1gwfjmtnfkxfaayoo3c_1602523931 is not permitted to log in\n","code":0,"duration":"1m31.286593268s"}
```

Comment:
This file is not like the others 😉

Reply:
<img src=https://media.giphy.com/media/COYGe9rZvfiaQ/giphy.gif height=150>